lib/clowne/adapters/sequel/associations/one_to_many.rb

```diff
@@ -7,12 +7,13 @@ module Associations
         class OneToMany < Base
           def call(record)
             clones =
-              with_scope.map do |child|
+              with_scope.lazy.map do |child|
                 clone_one(child).tap do |child_clone|
                   child_clone[:"#{reflection[:key]}"] = nil
                 end
-              end
-            record.remember_assoc(:"#{association_name}_attributes", clones)
+              end.map(&method(:record_wrapper)).to_a
```

Comment:
Why `to_a`?

And can we use only one `.map`? And, maybe, `yield_self`/`then`:

```ruby
with_scope.map do |child|
  clone_one(child).tap do
    # ...
  end.then(&method(:record_wrapper))
end
```

Reply:
> Why to_a?

Because of `Lazy`)

> And, maybe, yield_self/then:

It would be nice, but we support ruby 2.3+ :(

Reply:
> but we support ruby 2.3+ :(

You should have learned this-[Refinements](https://github.com/palkan/action_policy/blob/master/lib/action_policy/ext/yield_self_then.rb) 🧙‍♂️!

Reply:
Btw, we should add jRuby 9.2.5.0 to the build matrix (if want to support it); it has some Refinements-related regressions(

Reply:
You are crazy about refinements) Check it: https://github.com/palkan/clowne/pull/24